### ISSUE TITLE
Camera shadow

### ISSUE THREAD

**Issue by pedropro** [unrealcv/unrealcv#137]
- Operating System: Ubuntu 16.04
- UE4 Version: 4.19.1 (compiled from source)
- UnrealCV Version:
- Client (python2, 3 or matlab): python 2
- Problem Description: 

I noticed that on the images given by vget command, the camera/0/lit itself casts a circular shadow on objects. However this is now observed on my UE4 viewport.

To reproduce the problem, try to disable indirect light from the GlobalPostProcessVolume and place the camera between the light source and a surface.

Is there a way to disable the camera shadow? I still want shadows just not from the camera.

Anyway, thanks for this amazing tool.

**Comment by marksy-91** [comment]
Have you tried disabling shadowing for the Default Player/Pawn Class that you're using? I haven't experienced this issue but if you open the Lighting Settings of the Actor you should be see a field for 'Cast Shadow' which you can un-tick.

**Comment by pedropro** [comment]
Yes, I ended up doing that for a light source. Definitely not ideal as it switches off shadows cast by other objects. But thanks.

**Comment by AhmedAzizYedes** [comment]
I know this is a bit late but I think the problem is due to the DefaultPawn used in your GameMode.
I think that your GameMode is set to none and by default when the gameMode is set to none Unreal use the DefaultPawn which have a spherical mesh that casts shadows, try and switch it to **UE4CVGameMode**, they use their own pawn without mesh nor collisions.
Hope that solves your problem.

**Comment by Buffyqsf** [comment]
This is a little bit late too. But I am doing the similar things, and I found that you don't need to switched off shadows of the light source. Switch off the "cast shadows" of the two objects that you don't want to have shadows can work too, and this will only effect shadows between those two things.
